torch/csrc/distributed/rpc/init.cpp

```diff
@@ -214,6 +214,78 @@ PyObject* rpc_init(PyObject* /* unused */) {
                   If the current node is the owner, returns a reference to the
                   local value. Otherwise, throws an exception.
               )")
+          .def(
```

Comment:
Yes, one alternative is making this `RRefBase` and then inherit it in Python and create a Python `RRef` class. But if we go with that approach, we would need to let `C++` `pyRemote*` load and return the Python version of `RRef`. This would also need some minor modification to serde.  

another alternative is instead of inheriting, we can let the Python `RRef` holds an instance of the C++ `RRefBase`. I am not sure if this a clean solution. 

The main reason for choosing the current implementation is for simplicity, trying to keep all Python RRef APIs in one place. Let me know if you prefer to change it to inheritance or composition.